deno.json

```diff
@@ -0,0 +1,5 @@
+{
+  "workspace": {
+    "members": ["supabase/functions"]
```

Comment:
This is necessary to be able to mix Deno and npm workspaces in order to import the `Database` type from `@postgres-new/supabase` in our Edge Functions.